app/deepseg.cc

```diff
@@ -646,7 +682,15 @@ int main(int argc, char* argv[]) try {
 			// - default green (initial value)
 			bool canBlur = false;
 			if (pbk) {
-				if (grab_background(pbk, capGeo.value().first, capGeo.value().second, bg)<0)
+				int tw,th;
+				if ( crop_region.height ) {
+					tw = crop_region.width;
+					th = crop_region.height;
+				} else {
+					tw = capGeo.value().first;
+					th = capGeo.value().second;
+				}
```

Comment:
Should use `std::tie` as shown above.

Reply:
I think we will have the same problems as for the previous not committed suggestions.